content/02.body.md

```diff
@@ -1,13 +1,23 @@
 ## Introduction
 
-Networks contain information about relationships between entities ("edges between nodes").
-A node's degree is the number of relationships it has in the network.
+Networks contain information about relationships between entities (referred to here as "edges" between "nodes").
+A node's degree is the number of edges it has in the network.
+`TODO: do we need to discuss in-degree vs out-degree and hetnet degree here?`
```

Comment:
I think the discussion in the figure description should be sufficient for the introduction